package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "medic-conf",
-  "version": "3.1.0",
+  "version": "3.1.1",
```

Comment:
Let's leave the version for now. There are a bunch of other changes recently merged and I'm planning to bump to 3.2.0 and release soon anyway.

Reply:
This is weird. Master is actually on 3.1.1
https://github.com/medic/medic-conf/blob/master/package.json#L3

No idea why Github is highlighting this as a change.